apps/total-typescript/src/ability/ability.ts

```diff
@@ -32,7 +32,7 @@ export const createAppAbility = createMongoAbility as CreateAbility<AppAbility>
 type ViewerAbilityInput = {
   user?: User
   subscriber?: any
-  lesson?: Exercise
+  lesson?: LessonResource
```

Comment:
`LessonResource` is considerably less opinionated at the type level with no concerns regarding the underlying **media** (stackblitz, mux, etc)

Reply:
we have an existing LessonResource

https://github.com/skillrecordings/products/blob/main/packages/types/src/index.ts#L50

It's very strictly tied to the egghead oriented sellable data model

that entire package could probably be deprecated